src/components/Homepage/HomepageInsightCard.tsx

```diff
@@ -73,7 +73,7 @@ export const HomepageInsightCard = ({
         <InsightCard>
           You have{' '}
           {maybePluralize(actionItemInsight.ruleIds.length, 'action item')} due
-          by {moment(actionItemInsight.targetDate).local().format('MMM Do')}
+          by {moment.utc(actionItemInsight.targetDate).format('MMM Do')}
```

Comment:
```suggestion
          by {moment.utc(actionItemInsight.targetDate).local().format('MMM Do')}
```

Reply:
But isn't that just more confusing if we don't show the time?

Reply:
We have customers in NZ and other places far from UTC -- I think always displaying UTC values is more confusing unless we explicitly indicate UTC and/or add a timestamp.

Reply:
But that means the date may be different from the app before we change it there, because it will be shifted to match the users TZ.